### PR TITLE
Add radius and modify card style

### PR DESCRIPTION
**Pull request by inquisitiv-e**
- Map search results are biased towards vancouver
- Card component style change
- Sticky search bar 
Screen Shot 2019-06-30 at 2 38 19 PM

### CODE REVIEW

File: imports/ui/components/Card.jsx [hunk 1]
```diff
@@ -42,18 +45,40 @@ class CardComponent extends Component {
     return (
       <div>
         <Card className={classes.card}>
+          {/* Food Description */}
+
+          <CardHeader
+            avatar={
+              <Avatar aria-label="Recipe" className={classes.avatar}>
```
Comment (nicolehli) [inline imports/ui/components/Card.jsx]: Recipe?

File: imports/ui/components/Card.jsx [hunk 2]
```diff
@@ -41,19 +43,36 @@ class CardComponent extends Component {
     const { classes } = this.props;
     return (
       <div>
-        <Card className={classes.card}>
+        <Card className={classes.card} align="center">
+          {/* Food Description */}
+
+          <CardHeader
+            avatar={
+              <Avatar aria-label="Recipe" className={classes.avatar}>
```
Comment (nicolehli) [inline imports/ui/components/Card.jsx]: Recipe?

File: imports/ui/components/Card.jsx [hunk 3]
```diff
@@ -67,60 +86,34 @@ class CardComponent extends Component {
...
               </IconButton> */}
+
+            {/* Food Details */}
+
+            <div className={classes.price}>
+              <Typography variant="h6">{this.state.data.price} lb</Typography>
             </div>
-          </div>
+          </CardActions>
           {/* Food Image */}
```
Comment (nicolehli) [inline imports/ui/components/Card.jsx]: Unnecessary comment left over

File: imports/ui/components/Card.jsx [hunk 4]
```diff
@@ -67,60 +86,34 @@ class CardComponent extends Component {
...
-          <CardMedia
-            className={classes.image}
-            image={`/images/` + this.state.data.name + `.png`}
-          />
           {/* Food Details */}
```
Comment (nicolehli) [inline imports/ui/components/Card.jsx]: Unnecessary comment left over

**Comment by YaoLiu93** [comment]
These cards are a bit too big. The image is not the centerpiece, so definitely agree with @nicolehli's comment. 

The radius was included in a separate PR. Is that still applicable for this PR? If not, please update the title and PR description.

**Pull request closed without merging**